dip-README.md

```diff
@@ -0,0 +1,36 @@
+# ChatDIP
+Welcome to the Digital Power AI chatbot solution based on this [Azure sample repository](https://github.com/Azure-Samples/azure-search-openai-demo).
+
+Since the authors of the Azure sample project have already documented almost all you need to know, this README is meant to navigate quickly to the relevant parts.
```

Comment:
*to navigate to the relevant parts quickly 
of 
*to quickly navigate to the relevant parts